ansible/roles/docker/templates/alpine311.Dockerfile.j2

```diff
@@ -42,7 +42,7 @@ RUN apk add --no-cache --virtual .build-deps \
         libtool \
         autoconf
 
-RUN pip3 install tap2junit
+RUN pip3 install tap2junit==0.1.6
```

Comment:
We could maybe use a variable so we don't have to update every template when we update in the future. 
To do this:
1. Add a new variable to `ansible/roles/docker/vars/main.yml` for the tap2junit version.
2. Replace in the templates using handlebars, e.g.
```suggestion
RUN pip3 install tap2junit=={{ tap2junit_version }}
```